pkg/autodiscovery/providers/tests/advanced_ad.yaml

```diff
@@ -0,0 +1,9 @@
+advanced_ad_identifiers:
+  - kube_service:
+      name: svc-name
+      namespace: svc-ns
+
+init_config:
+
+instances:
+  - foo: bar
```

Comment:
Note: not sure if this is placeholder, but flagging just in case.

Reply:
This is just a test file for the agent internals and will not be exposed to users.